static/app/types/profiling.d.ts

```diff
@@ -54,12 +54,17 @@ declare namespace Profiling {
   // This extends speedscope's schema - we are keeping this as is, but we are likely to diverge as we add more
   // sentry related features to the flamegraphs. This should happen after the MVP integration
   type Schema = {
-    name: string;
-    activeProfileIndex?: number;
-    duration_ns: number;
+    durationNS: number;
+    platform: string;
+    profileID: string;
     profiles: ReadonlyArray<ProfileTypes>;
+    projectID: number;
     shared: {
       frames: ReadonlyArray<Omit<FrameInfo, 'key'>>;
     };
+    transactionName: string;
+    version: string;
+    activeProfileIndex?: number;
+    androidClock?: 'Global' | 'Dual' | 'Wall' | 'Cpu';
```

Comment:
Nice 👍🏼. @phacops does "dual" indicate that we should disable the sync transaction feature here?

Reply:
On a separate not. Why dont we just call this `clock`? Do we lack the info from iOS on what clock it uses maybe?

Reply:
`Cpu` indicates that we should disable the sync transaction feature.

Reply:
`Dual` indicates we have `Cpu` and `Wall` times available and we default to `Wall` in that case. It's called `androidClock` because we don't have this info for `iOS` and just assume wall time there.